src/Serilog.Sinks.PeriodicBatching/Sinks/PeriodicBatching/PortableTimer.cs

```
@@ -109,5 +115,13 @@ public void Dispose()
             }
         }
     }
+
+    static class TaskExtensions
+    {
+        public static async void AsObserved(this Task task)
+        {
+            await task.ConfigureAwait(false);
+        } 
+    }
```

This retains fail fast behavior if handler throws